tests/std/tests/GH_000545_include_compare/test_ranges.cpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <version> // TRANSITION, EDG support for concepts

```

Comment:
I'd prefer this to be `// TRANSITION, GH-395`, but it's probably not worth the trouble to make just that change now.